components/core/config/config.ts

```diff
@@ -329,6 +331,7 @@ export interface TreeConfig {
   nzBlockNode?: boolean;
   nzShowIcon?: boolean;
   nzHideUnMatched?: boolean;
+  nzVirtualItemSize?: number;
```

Comment:
You have made `nzVirtualItemSize` global for tab's config, but never changed tab component.